pkg/logger/global.go

```diff
@@ -0,0 +1,139 @@
+package logger
+
+// Initialize the logger at package level.
+var (
+	//nolint:gochecknoglobals
+	logger *Logger
+)
+
+// InitializeGlobal initializes the global logger at package level.
+// Once initialized, the logger can be used by calling the directly the classic logging functions (Debug, Info...).
+func InitializeGlobal(path string) error {
+	var err error
+
+	logger, err = New(path)
+
+	return err
+}
+
+// Debug logs a message at the debug level.
+// This function takes a variable number of arguments and uses fmt.Sprint to construct the log message.
+// Note: The global logger must be initialized before calling this function, otherwise it will panic.
+func Debug(args ...interface{}) {
+	if logger == nil {
+		panic("global logger not initialized")
```

Comment:
this string is repeated many time, let's put it into a constant variable at the beginning of the file.

Reply:
should be fixed